helm_deploy/visit-scheduler/values.yaml

```diff
@@ -23,8 +23,7 @@ generic-service:
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
     BOOKING_NOTICE_PERIOD_MAXIMUM_DAYS: 28
     HMPPS_SQS_USE_WEB_TOKEN: "true"
-    # TODO: Update to 28 post performance analysis
-    FLAG_VISITS_NUMBER_OF_DAYS_AHEAD: 21
+    FLAG_VISITS_NUMBER_OF_DAYS_AHEAD: 28
```

Comment:
updating number of days ahead to 28 after analysing performance over the weekend run.